packages/core/src/directory.ts

```diff
@@ -57,7 +57,7 @@ export class DirectorySync implements DirectorySubscriber {
 
             org.members.push(newOrgMember);
 
-            await this.server.updateMetaData(org);
+            await this.server.updateMetaData(org, "directory-sync");
```

Comment:
There's no real `accountId` making this change, so we use this special string. Seemed fine as it is, but I can make it a constant if you prefer.